## 01_fix_make_distclean.patch
## by Mike Kelly <pioto@pioto.org>
##
## Fix `make distcheck`

--- a/luks/Makefile.am
+++ b/luks/Makefile.am
@@ -2,7 +2,7 @@ moduledir = $(libdir)/cryptsetup
 
 noinst_LTLIBRARIES = libluks.la
 
-libluks_la_CFLAGS = -I sha -Wall
+libluks_la_CFLAGS = -I $(top_srcdir)/luks/sha -Wall
 
 libluks_la_SOURCES = \
 	af.c \
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -15,7 +15,7 @@ cryptsetup_SOURCES = \
 
 cryptsetup_LDADD = \
 	@POPT_LIBS@				\
-	$(top_srcdir)/lib/libcryptsetup.la
+	$(top_builddir)/lib/libcryptsetup.la
 
 if STATIC_CRYPTSETUP
 _STATIC_FLAGS = -all-static
